BE/src/main/java/com/sidedish/api/categories/CategoryController.java

```diff
@@ -37,16 +45,31 @@ public void setAsText(String text) {
     }
 
     @GetMapping("/{type}")
-    public CollectionModel<ItemResource> getCategoriesByType(@PathVariable CategoryType type, @RequestParam Long pageId) {
+    public CollectionModel<ItemResource> getItemsByCategory(@PathVariable CategoryType type, @RequestParam Long pageId, @RequestParam(defaultValue = "4") int pageCount) {
 
-        List<Item> items = itemService.findUnitPageById(type.getTypeNumber(), pageId);
+        List<Item> items = itemService.findUnitPageById(type, pageId, pageCount);
         List<ItemResource> itemResources = items.stream().map(ItemResource::new).collect(Collectors.toList());
 
         CollectionModel<ItemResource> responseMainType = CollectionModel.of(itemResources);
-        responseMainType.add(linkTo(methodOn(CategoryController.class).getCategoriesByType(type, pageId)).withSelfRel());
-        responseMainType.add(linkTo(methodOn(CategoryController.class).getCategoriesByType(type,pageId-1)).withRel("prev-page"));
-        responseMainType.add(linkTo(methodOn(CategoryController.class).getCategoriesByType(type, pageId+1)).withRel("next-page"));
+        responseMainType.add(linkTo(methodOn(CategoryController.class).getItemsByCategory(type, pageId, pageCount)).withSelfRel());
+        responseMainType.add(linkTo(methodOn(CategoryController.class).getItemsByCategory(type,pageId-1, pageCount)).withRel("prev-page"));
+        responseMainType.add(linkTo(methodOn(CategoryController.class).getItemsByCategory(type, pageId+1, pageCount)).withRel("next-page"));
 
         return responseMainType;
     }
+
+    @GetMapping("/{type}/detail")
+    public List<ResponseItemDto> getItemByDetailType(@PathVariable String type) {
+        List<Item> findItems = itemService.findItemByDetailType(type);
```

Comment:
categoryType과 혼동의 여지가 있어서 구분해서 사용하는게 좋겠습니다.

```suggestion
    public List<ResponseItemDto> getItemByDetailType(@PathVariable(name = "type") String detailType) {
        List<Item> findItems = itemService.findItemByDetailType(detailType);
```